src/cli.rs

```diff
@@ -229,6 +261,13 @@ impl<'a> Cli<'a> {
             config.no_ignore_vcs
         };
 
+        config.for_each_fn = match self.streaming {
+            Some(Streaming::Json) => Some(|l: LanguageType, e| println!("{{language: '{}', stats: '{}'}},", l.name(), serde_json::json!(e))),
```

Comment:
```suggestion
            Some(Streaming::Json) => Some(|l: LanguageType, e| println!("{}", serde_json::json!({ "language": l.name(), "stats": e  }))),
```

Reply:
For the "simple" output, it prints a row per file. If you think it is useful to include a header line, I can add it with a leading "#" so that the batch process could ignore it. Let me know.

Reply:
Not sure if the default `--streaming=none` should be kept in the command line options, if you think it does no harm, I will consider the implementation being completed :-)

Reply:
Well I don't think there needs to be a none option. None is implied by it not being present in the command.

Reply:
Done !

Reply:
If OK, is it possible to merge this into the trunk, or if you would like to add any more test cases?